app/assets/stylesheets/spree/frontend/solidus_social.css

```diff
@@ -1,4 +1,4 @@
 /*
- *= require spree/frontend
+ *= require spree/frontend/fontello
```

Comment:
Could you explain this one, please?

Reply:
Hi @kennyadsl I faced an issue after installing the extension i.e - somehow the sprockets is looking for the file `spree/frontend` to be there, and raised an error with Sprockets::FileNotFound. I have to change this so that it points to an file instead of frontend directory as Sprockets is unable to find. 

Here is the screenshot: 
![Screenshot-from-2024-12-24-21-56-42-12-24-2024_10_15_PM](https://github.com/user-attachments/assets/5e26c422-419a-45c3-bbad-0db0262c82d3)

Let me know if there any better way to handle this.